lib/kitchen/element_base.rb

```diff
@@ -429,6 +429,12 @@ def search_with(*enumerator_classes)
     def cut(to: nil)
       block_error_if(block_given?)
 
+      temp = clone
```

Comment:
In `paste`, a `clone` is needed because we are changing the IDs on line 474.  Here we aren't changing this element so we don't need (or want) a clone.  You can simply do 

```ruby
raw.traverse do |node|        
  next if node.text? || node.document?        
  document.record_id_cut(node[:id])      
end
```